packages/ui/components/paypal-settle-button/index.tsx

```diff
@@ -91,7 +94,7 @@ class PayPalSettlementButton extends Component<Props, State> {
 
   async handlePayPalPayment() {
     try {
-      let cleanAmount = Number(this.props.displayAmount.replace(/[^0-9\.\,]/g, ''))
+      let cleanAmount = Number(this.props.displayAmount.replace(/[^0-9\.,]/g, ''))
```

Comment:
This did not fix the problem I was seeing so I don't know if it's necessary or not, probably...

Reply:
there isn't a difference between these two regular expressions